config/samples/capsule_v1alpha1_capsuleconfiguration.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: capsule.clastix.io/v1alpha1
+kind: CapsuleConfiguration
+metadata:
+  name: default
+spec:
+  userGroups: ["capsule.clastix.io"]
+  forceTenantPrefix: false
+  protectedNamespaceRegex: ""
+  allowTenantIngressHostnamesCollision: false
```

Comment:
it should be `allowTenantIngressHostnamesCollision: false` according to #207

Reply:
I'd like to address this in a separate PR, referencing this issue since it would require some test changes and wanted to avoid making a fat PR.